pulsar-broker/src/test/java/org/apache/pulsar/client/api/RetryTopicTest.java

```diff
@@ -457,4 +462,58 @@ public void testRetryTopicByCustomTopicName() throws Exception {
         checkConsumer.close();
     }
 
+
+    @Test
```

Comment:
```suggestion
    @Test(timeOut = 30000L)
```

Since the original failure is hanging, we should add a timeout field here.

Reply:
done